src/components/CollectionsAndPalettes/Collections/Collections.tsx

```diff
@@ -41,7 +53,11 @@ export default class Collections extends React.Component<any, State> {
     return (
       <React.Fragment>
         <div className={styles.cardCollection}>
-          <UploadCard caption={"Create a Collection"} hasIcon />
+          <UploadCard
+            caption={"Create a Collection"}
+            hasIcon
+            showModal={this.createCollection}
```

Comment:
I think this would be better named 'onClick', as we might want to do something else instead of opening a modal down the line.